torch/nn/modules/conv.py

```diff
@@ -23,9 +23,9 @@ class _ConvNd(Module):
 
     _in_channels: int
     out_channels: int
-    kernel_size: Tuple[int, ...]
-    stride: Tuple[int, ...]
-    padding: Tuple[int, ...]
+    kernel_size: Any
+    stride: Any
+    padding: Any
```

Comment:
Can you please explain why? `Tuple` or `Sequence` sounds like a right type here

Reply:
Tuple is the 'correct' type here, the issue with setting this to Tuple is it mismatches with _output_padding, which is typed with a List[int]. We can set this to Tuple[int, ...] and cast every call to _output_padding with List[int], but this unfortunately fails when scripting. Typing with Any avoids conflict with the signature of _output_padding